catalyst_voices/uikit_example/lib/examples_list.dart

```diff
@@ -24,6 +25,11 @@ class ExamplesListPage extends StatelessWidget {
         route: VoicesSnackbarExample.route,
         page: VoicesSnackbarExample(),
       ),
+      ExampleTile(
+        title: 'Voices Widgets',
+        route: VoicesWidgetsExample.route,
+        page: VoicesWidgetsExample(),
```

Comment:
Should it be an example just for segmented buttons or you want to add more stuff there? Widgets example sounds very generic, could it be more specific?

Reply:
Maybe `VoicesButtonsExample`?

Reply:
I'm on the fence about it. I built this screen as more generic page for displaying all kind of widgets just because there will be many of them and this way it may be faster to implement.

On the other side it may be less readable after while.

What do you think @dtscalac @minikin? Should we stick with page per widget approach?

Reply:
In the future when we have many widgets I wanted to add a search field to the list page so that you can search what you need. If create sections and subsections then this search wouldn't make sense. 

You can extract the common staff that is used for example_pages and reuse it if there's too much boilerplate

Reply:
Gotcha, makes sense. I'll stick with "Page per Widget" approach in that case.